components/en/main_nav.html

```diff
@@ -1,17 +1,27 @@
 <!-- Each of these li (list item) elements represents a button in our menu -->
 <li onclick="location.href='{fill_parents_html}en/index.html'" class="nav-elem">
-    <div class="nav-elem-text">Main Page</div>
+    <div class="nav-elem-text">Home</div>
 </li>
-<li onclick="location.href='#'" class="nav-elem">
-    <div class="nav-elem-text">About Us</div>
+<li class="nav-elem dropdown closed">
+    <div class="nav-elem-text sub-menu-btn closed">About Us</div>
+    <div class="sub-menu block container">
+        <ul class="sub-menu list flex column">
+            <li id="kindergarten" onclick="location.href='{{kindergarten.link}}'">Kindergarten</li>
+            <li id="primary school" onclick="location.href='{{primary_school.link}}'">Primary School</li>
+            <li id="high school" onclick="location.href='{{high_school.link}}'">High School</li>
+        </ul>
+    </div>
 </li>
-<li onclick="location.href='#'" class="nav-elem">
+<!-- <li onclick="location.href='#'" class="nav-elem">
```

Comment:
let's just remove it then? What's the point of it being commented out?

Reply:
Changed it in the other PR